src/calculateRentalCost.js

```diff
@@ -4,7 +4,13 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
+  } else if (days >= 3) {
+    return days * 40 - 20;
```

Comment:
Replace the literal '3' with a constant that explains its meaning, such as 'SHORT_TERM_RENTAL_DAYS'. Similarly, replace '20' with 'SHORT_TERM_DISCOUNT'.